lib/screens/home.dart

```diff
@@ -69,6 +69,36 @@ class HomeState extends State<Home> with SingleTickerProviderStateMixin {
         // backgroundColor: lightGreen,
         centerTitle: true,
         title: Text(widget.appName),
+        actions: [
+          IconButton(
+            icon: const Icon(Icons.info),
+            onPressed: () {
+              showAboutDialog(
+                context: context,
+                applicationName: 'Key Pod',
+                applicationVersion: '1.2.3',
```

Comment:
This version number is different from the one in `pubspec.yaml`, is this intentional?

Reply:
Yes, I need some code to get the actual version number - a task for another day.

Reply:
We have a [function in solidpod](https://github.com/anusii/solidpod/blob/494d372fbea2e3e29835df34da7b98335cb40da6/lib/src/solid/api/rest_api.dart#L626) to do this. Just did that here.

Reply:
Excellent. Thanks for adding that in @cdawei.